src/workflows/run_benchmark/config.vsh.yaml

```
@@ -42,12 +42,24 @@ argument_groups:
         required: true
         direction: output
         default: task_info.yaml
-  - name: Methods
+  - name: Include methods
     arguments:
```

Can add a description to the argument group.

```suggestion
  - name: Method filtering
    description: Use these arguments to filter methods by name. By default, all methods are run. If `--methods_include` is defined, only those methods are run. If `--methods_exclude` is defined, all methods except those specified are run. These arguments are mutually exclusive, so don't provide both `--methods_include` and `--methods_exclude`.
    arguments: